app/views/contexts/summary/group.py

```diff
@@ -77,10 +93,62 @@ def _build_blocks(
                     ]
                 )
 
+            else:
+                section = schema.get_section(location.section_id)
+
+                summary_item = self.get_summary_item_for_list_for_section_summary(
+                    summary=section.get("summary", {}),
+                    list_name=block.get("for_list", {}),
+                )
```

Comment:
This can be dropped in favour of the method in questionnaire schema: `get_summary_item_for_list_for_section`

Reply:
Using method from questionnaire schema now 👍